bin/cli.js

```diff
@@ -187,8 +188,14 @@ if (config.writeToStdout) {
   }
 } else {
   console.log('Writting output to following files...');
-  for (const [file, data] of output) {
-    fs.writeFileSync(file, data);
-    console.log(file);
-  }
+  common.mkdirp(config.outputDir, err => {
```

Comment:
Will this work correctly for `config.outputDir === '.'`?

Reply:
Yes, why it shouldn't?

Reply:
Ok then, just making sure.